### ISSUE TITLE
UserWarning: "`categorical_crossentropy` received `from_logits=True`, but the `output` argument was produced by a Softmax activation and thus does not represent logits. Was this intended?

### ISSUE THREAD

**Issue by ankhanh-vni** [VisiumCH/dvc-workshop#10]
Should we modify the from_logits parameters in dvc_workshop/models/classifier.py to False, since we are passing a softmax activation function at the output layer after all?

**Comment by TreeOfEast** [comment]
> Should we modify the from_logits parameters in dvc_workshop/models/classifier.py to False, since we are passing a softmax activation function at the output layer after all?

I encountered this issue while using ipynb, which caused a direct interruption of my kernel. I don't know how you solved this problem?

**Comment by ankhanh-vni** [comment]
I set the parameter *from_logits* to False. Hope it helps
